libs/clients/national-registry/v2/src/lib/types/religion.dto.ts

```diff
@@ -0,0 +1,46 @@
+import { Kodar } from '../../../gen/fetch'
+
+export interface ReligionDto {
+  code: string
+  name: string | null
+}
+
+const order =
```

Comment:
There is similar code in multiple places in the repo now. Please move/refactor this to libs/shared/utils.